src/mtev_cluster.c

```diff
@@ -65,13 +82,16 @@ struct mtev_cluster_t {
   mtev_cht_t **cht;
   mtev_net_heartbeat_ctx *hbctx;
   mtev_cluster_node_t *oldest_node;
+  struct jl_array_list *hb_payloads;
```

Comment:
I think ti is very awkward to use an internal json list implementation here.  Can't we just use a normal C array? or an mtev_hash ?